Homework/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	<body>
+
```

Comment:
@JsonLitz I typically will leave a single empty line between sections of my html. This will help me visualize what's going on a bit better. I will say that proper indentation would be more of a concern than the empty lines, so you were right in focusing on the indentation. Again, none of these things will break your code, but we just want to try to make it look as clean as possible ;)